routing/probability_estimator.go

```diff
@@ -80,19 +80,20 @@ func (p *probabilityEstimator) getNodeProbability(now time.Time,
 	totalWeight := aprioriFactor
 
 	for _, result := range results {
-		age := now.Sub(result.Timestamp)
-
 		switch {
+
 		// Weigh success with a constant high weight of 1. There is no
-		// decay.
-		case result.Success:
+		// decay. Amt is never zero, so this clause is never executed
+		// when result.SuccessAmt is zero.
+		case amt <= result.SuccessAmt:
 			totalWeight++
-			probabilitiesTotal += p.prevSuccessProbability
+			probabilitiesTotal += prevSuccessProbability
```

Comment:
how come this is not `p.prevSuccessProbability`? I don't see where this value is coming from.

Reply:
I think a rebase mistake, fixed